src/lib/services/email-service.ts

```diff
@@ -138,6 +140,86 @@ export class EmailService {
         });
     }
 
+    async sendScheduledChangeConflictEmail(
+        recipient: string,
+        conflictScope: 'flag' | 'strategy',
+        changeRequests: {
+            id: number;
+            scheduledAt: string;
+            title?: string;
+        }[],
+        strategyIdOrFlagName: string,
+    ) {
+        if (this.configured()) {
+            const year = new Date().getFullYear();
+            const conflict =
+                conflictScope === 'flag'
+                    ? `The feature flag ${strategyIdOrFlagName} has been archived.`
```

Comment:
We can't link to a deleted strategy (because that UUID is gone), but should we link to an archived flag? It might have been deleted in the meantime (and a new one might have been created), but if we want to make it as easy as possible for people to fix the conflict (by reviving the flag), then a link to the flag would be very useful.

Reply:
I think linking to the change requests is enough imo